### DISCUSSION TITLE
Not working on Edge 17/18 ?

### DISCUSSION THREAD

**Discussion by Mushr0000m** [alpinejs/alpine#3210] [7. Bugs (archived)]
I had the impression that Alpine V3 was working on Edge before chromium (15/16/17/18) but I got an error `Expected identifier, string or number`, like in this old issue : https://github.com/alpinejs/alpine/issues/109

The bug is present on the alpine website visible here : https://alpinejs.dev/component/modal I tested it using browserstack

**Comment by ekwoka** [comment]
Well that old one was before V3.

It would be helpful if you could use an unminified version to see where the error is actually coming from (like what line)

**Comment by Mushr0000m** [comment]
It come from this (exactly `...property, ` line) : https://github.com/alpinejs/alpine/blob/cf23ec88076fc746bcb95e5843ad3bd5843c17f0/packages/alpinejs/src/scope.js#L77-L81 

FYI spread operator works on Edge

**Reply by ekwoka** [reply]
https://caniuse.com/mdn-javascript_operators_object_initializer_spread_properties

Spread operator for objects does not.

I believe when making your own bundle, you should be able to specify an older standard to transpile for and it would handle this for you.

So that's what is recommended.

Or make a PR changing such instances to something like 
```js
Object.assign(property, { get: getter, set: setter })
```

**Reply by Mushr0000m** [reply]
Haha didn't see that difference for objects 🤦🏻‍♂️. Sorry about that.
But just to be clear what browser version is officialy supported by Alpine ?
